README.md

```diff
@@ -1,2 +1,82 @@
 # velero-plugin
-HPE plugin for Velero
\ No newline at end of file
+
+HPE plugin for Velero.
+To take snapshots of HPE volumes through velero you need to install and configure the HPE Snapshotter plugin.
+
+## Installation
+
+Refer to <https://velero.io/docs/v1.1.0/get-started/> for installing velero.
+
+### 1. Create VolumeSnapshotLocation CRD for HPE Snapshotter
+
+```markdown
+
+apiVersion: velero.io/v1
+kind: VolumeSnapshotLocation
+metadata:
+  name: hpe-csp
+  namespace: velero
+spec:
+  provider: hpe.com/snapshotter
+  config: {
+       "secret-name": "nimble-secret",
+       "secret-namespace": "kube-system"
+  }
+```
+
+### 2. Create secret to communicate with HPE Nimble Storage CSP Service
+
+```markdown
+
+https://raw.githubusercontent.com/hpe-storage/csi-driver/master/examples/kubernetes/secret.yaml
```

Comment:
Can you add the actual secret here and add some verbiage around it? The 's' in Service shouldn't be capitalized.
I also think step #2 and #3 should be collapsed for readability.
So,

### 2. Deploy a Container Service Provider service
In order for the snapshotter to perform snapshots, it needs to communicate with a CSP. Currently only HPE Nimble Storage provides a CSP and the rest of the installation assumes Nimble.

Create a CSP secret that maps to a Nimble array management IP address and "Power User" (or Administrator):
```
apiVersion: v1
kind: Secret
metadata:
  name: nimble-secret
  namespace: kube-system
stringData:
  serviceName: nimble-csp-svc
  servicePort: "8080"
  backend: 192.168.1.1
  username: admin
data:
  # echo -n admin | base64
  password: YWRtaW4=
```
Deploy the CSP (it will be deployed in the 'velero' namespace on the cluster):
```
kubectl create -f https://raw.githubusercontent.com/hpe-storage/velero-plugin/master/nimble-csp.yaml
```

Reply:
Thanks will update